src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1232,17 +1232,20 @@ internal enum ErrorCode
         ERR_FixedBufferTooManyDimensions = 7092,
         ERR_CantReadConfigFile = 7093,
         ERR_BadAwaitInCatchFilter = 7094,
-        WRN_FilterIsConstant = 7095,
+        // WRN_FilterIsConstant = 7095,   // replaced by FilterIsConstantFalse and FilterIsConstantTrue
```

Comment:
Sure, I will keep the ID value 7095, and use the new name `FilterIsConstantTrue`. 
My thinking was to keep related warnings close together, but I guess keeping the orignal warning ID for the sake of backward compatibility (for warning disable pragma) is more important.